library_list.php

```diff
@@ -115,9 +115,10 @@
 
 // Add js.
 $liburl = \mod_hvp\view_assets::getsiteroot() . '/mod/hvp/library/';
+$relpath = '/' . preg_replace('/^[^:]+:\/\/[^\/]+\//', '', $liburl);
 
-hvp_admin_add_generic_css_and_js($PAGE, $liburl, $settings);
-$PAGE->requires->js(new moodle_url($liburl . 'js/h5p-library-list.js' . hvp_get_cache_buster()), true);
+hvp_admin_add_generic_css_and_js($PAGE, $relpath, $settings);
+$PAGE->requires->js($relpath . 'js/h5p-library-list.js', true);
```

Comment:
I would still expect hvp_get_cache_buster() to be in the url, or something similar?

If the js changes then we want the url to change so its a fresh cache item

Reply:
Isn't that handled by /lib/javascript.php ?

Reply:
`lib/javascript.php` I think will just add the etag. Whereas we still want the URL to change each time so the cache gets updated - this was discussed below see https://github.com/catalyst/moodle-mod_hvp/pull/29/files#r1037659505

I think the appropriate action is to re-add back the cache buster

Reply:
No jason is correct, a purge cache is doing this correctly already:

![image](https://user-images.githubusercontent.com/187449/206367260-8e046ca8-d776-4d12-8221-3c64690ba249.png)